rust/perspective-viewer/src/rust/components/column_selector/filter_column.rs

```diff
@@ -456,6 +456,8 @@ impl Component for FilterColumn {
         }
         .unwrap_or_default();
 
+        let final_col_type = col_type.expect("Unknown column");
```

Comment:
There is no point in using `expect()` in Perspective WASM code because the error message does not propagate to the JS console. Generally we want to safely handle when this is missing because `panic!` leads to a frozen/broken web app that has to be refreshed.